adam/ontology/phase1_ontology.py

```diff
@@ -286,6 +242,13 @@ def above(obj1: SubObject, obj2: SubObject) -> Tuple[SubObjectRelation, ...]:
 _PERSON_SCHEMA_LEFT_LEG = SubObject(_LEG_SCHEMA)
 _PERSON_SCHEMA_RIGHT_LEG = SubObject(_LEG_SCHEMA)
 
+_PERSON_SCHEMA_LIMBS = [
```

Comment:
Nitpick: If you are going to include `HEAD` maybe prefer `APPENDAGES` to `LIMBS`?